includes/gateways/paypal/checkout-actions.php

```diff
@@ -219,6 +195,18 @@ function create_order( $purchase_data ) {
 			$api      = new API();
 			$response = $api->make_request( 'v2/checkout/orders', $order_data );
 
+			if ( ! isset( $response->id ) && _is_item_total_mismatch( $response ) ) {
+				// Try again without the item breakdown. That way if we have an error in our totals the whole API request won't fail.
```

Comment:
What would you think about adding an `edd_debug_log` here indicating that there was an item total mismatch and then logging the `$order_data`? That way we could see exactly what we were sending and maybe get some clues about where it could be fixed.

Reply:
I think that is a good idea @robincornett, these types of things are always hard to track down.

Reply:
I like the idea but might opt for a gateway payment error instead, just because it's a bit more persistent than the debug log.

Do you want to do it @robincornett or do you want me to?

Reply:
Sure, I can do it, since I can consistently get the error to happen.